src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Microsoft.Azure.Management.PolicyInsights.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.PolicyInsights</PackageId>
+    <Description>Provides policy insights operations for Microsoft Azure.</Description>
+    <AssemblyName>Microsoft.Azure.Management.PolicyInsights</AssemblyName>
+    <Version>1.0.0-preview</Version>
```

Comment:
@bulentelmaci if this is your v1 and you are planning to go preview, I would recommend you go as 0.9.0-preview.
Because when you are ready to go stable (non preview) you can then mark it as 1.0.0 and that will make your customers take notice on your first stable version of your SDK.
All V1 products go out as 0.9.0-preview

Reply:
Will do